### ISSUE TITLE
Add "you're in the wrong place" text to DSF contact page

### ISSUE THREAD

**Issue by freakboy3742** [django/djangoproject.com#406]
About 75% of people who use the Django contact form are using it to:

 * Report a bug in Django
 * Report a problem with the Django website
 * Ask for help debugging their site

and so on. There's clearly some confusion about what the contact page is for; it would be great to be able to push some of this traffic towards more appropriate destinations.

Putting some text at the top of the Contact the Foundation page might help:

"This contact form is for the Django Software Foundation - the legal and fundraising arm of the Django project. If you want to report a bug in Django, use the ticket tracker. If you want to report a problem with this website, use the github repo. If you've got questions about how to use Django, use the django-users mailing list"

**Comment by olasitarska** [comment]
Maybe one missing bit is: *If you want to report a bug in documentation, use trac?* I think this one is easily getting confused by people.

**Comment by bmispelon** [comment]
That's a good idea, I'll open a separate issue on it.

Thanks! :bee:
